test/pom.xml

```diff
@@ -212,6 +212,7 @@
           <configuration>
             <testSourceDirectory>${project.basedir}/src/main/java/</testSourceDirectory>
             <testClassesDirectory>${project.build.directory}/classes/</testClassesDirectory>
+            <skipTests>${failsafe.skip}</skipTests>
```

Comment:
This isn't needed. There's a separate `-DskipITs` that controls failsafe separately from surefire. Having it here will interfere with the ability to use system properties to skip, since configuration in the pom.xml overrides system properties.

Reply:
@ctubbsii - the reason for adding separate `failsafe.skip` and `surefire.skip` properties was so that surefire tests could be skipped but failsafe tests would still be run. `-DskipITs` allows you to run  surefire (unit) tests without failsafe (integration) tests while `-DskipTests` skips both surefire and failsafe test. I'll remove these for now to get this merged, but we might want to consider doing something like this in the future?

Reply:
@drewfarris I think this works to just run the ITs: ```mvn clean verify -Dtest=foo```